### PR TITLE
fix autoCancelAfterFailures in ModuleAPI

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR fixes the pipe (vertical bar) rendering in the `autoCancelAfterFailures` entry in the [Guides > Module API > Options](https://docs.cypress.io/guides/guides/module-api#Options) table. The `mdx` Markdown source was attempting to use the HTML Character Entity Number `&#124` for "vertical bar". The `|` character (which is a otherwise a table control character in Markdown) is escaped as `\|` instead for correct rendering.

~~Also, since `false` is a value, not a type, the type is changed to show `boolean` for consistency with the other table entries.~~

---
BEFORE
before autoCancelAfterFailures

---
AFTER
after autoCancelAfterFailures

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Preview change on https://deploy-preview-5628--benevolent-cat-040f48.netlify.app/guides/guides/module-api#Options

File: docs/guides/guides/module-api.mdx [hunk 1]
```diff
@@ -56,31 +56,31 @@ node e2e-run-tests.js
...
-| `testingType`             | _string_               | Specify the type of tests to execute; either `e2e` or `component`. Defaults to `e2e`                                                                                                                     |
+| Option                    | Type                  | Description                                                                                                                                                                                              |
+| ------------------------- | --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `autoCancelAfterFailures` | _number_ \| _boolean_ | Specify the number of failures to cancel a run being recorded to the Cloud or `false` to disable auto-cancellation.                                                                                      |
```
Comment (emilyrohrbough) [inline docs/guides/guides/module-api.mdx:61]:
```suggestion
| `autoCancelAfterFailures` | _number_ \| _false_ | Specify the number of failures to cancel a run being recorded to the Cloud or `false` to disable auto-cancellation.                                                                                      |
```

Comment (emilyrohrbough) [reply docs/guides/guides/module-api.mdx:61]: False is the correct option here if a number is not specified

Comment (MikeMcC399) [reply docs/guides/guides/module-api.mdx:61]: Thanks for the correction. I was unfamiliar with the concept of TypeScript literal types.
I will revert to using `_false_` as a Type, so the only correction in this PR will be the rendering of the pipe symbol.

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
Thanks Mike!

**Pull request merged**